cf-manifest/build_manifest.sh

```diff
@@ -4,10 +4,15 @@ set -e
 cd $(dirname $0)
 
 terraform_outputs=${TERRAFORM_OUTPUTS:-"outputs/terraform-outputs.yml"}
+secrets=${SECRETS:-"outputs/cf-secrets.yml"}
+ssl_certs=${SSL_CERTS:-"outputs/cf-ssl-certificates.yml"}
 
 spruce merge \
   --prune meta --prune lamb_meta \
   --prune terraform_outputs \
+  --prune secrets \
   deployments/*.yml \
   deployments/aws/*.yml \
-  ${terraform_outputs}
+  ${terraform_outputs} \
+  ${secrets} \
+  ${ssl_certs}
```

Comment:
@mtekel SSL certs are under the ````secrets```` key and merged, so they'll already be being removed.